### PR TITLE
Add TLS metrics support

### PR DESCRIPTION
**Pull request by dmaganto**

### CODE REVIEW

**Comment by SuperQ** [comment]
This should use the [prometheus/exporter/toolkit](https://github.com/prometheus/exporter-toolkit) https package.

**Comment by dmaganto** [comment]
@SuperQ modified to use exporter-toolkit.
Regards

File: cmd/process-exporter/main.go [hunk 1]
```diff
@@ -306,6 +308,8 @@ func main() {
 			"print manual")
 		configPath = flag.String("config.path", "",
 			"path to YAML config file")
+		tlsConfigFile = flag.String("web.config", "",
```
Comment (SuperQ) [inline cmd/process-exporter/main.go:311]:
```suggestion
		tlsConfigFile = flag.String("web.config.file", "",
```

**Review by SuperQ — COMMENTED** [review]
Please also squash the commits.

File: cmd/process-exporter/main.go (hunk 1 above)
Comment (dmaganto) [reply cmd/process-exporter/main.go]: Done squash and change in line 311.

**Review by SuperQ — APPROVED** [review]
LGTM

**Comment by tuannh99** [comment]
@SuperQ Hi. How's the plan to merge/release this?

**Comment by ncabatoff** [comment]
I'm open to merging this PR, if I understand correctly what it does, but I'd like to see some doc changes included first.

**Comment by dmaganto** [comment]
Sure, @ncabatoff let me do it. This is to enable TLS in the /metrics endpoint in order to allow use this in secure environment.

**Comment by lucian-vanghele** [comment]
@dmaganto  - I believe @ncabatoff is waiting to update documentation also. Just pointing to Prometheus documentation https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md should be enough I would say.

**Comment by dmaganto** [comment]
Yes, you're right. Do you think that should be enough @ncabatoff?

**Comment by ncabatoff** [comment]
Yeah that would be fine with me, if you mean adding a change to this PR updating the readme to document the new flag and to reference exporter-toolkit's docs re the file format.

**Comment by dmaganto** [comment]
@ncabatoff I added some documentation and reference for further information to the main project [repo](https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md). Please take a look.

**Review by ncabatoff — APPROVED** [review]

**Comment by ncabatoff** [comment]
Thanks!

**Pull request merged**